src/Plutarch/Internal/Builtin.hs

```diff
@@ -0,0 +1,810 @@
+{-# LANGUAGE FlexibleInstances #-}
```

Comment:
This is quite massive. More so than I thought, would it be better to make `Plutarch.Builtins.XYZ` modules instead?

Reply:
Yeah, I agree.